rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -33,14 +33,15 @@
 #include "rmw/allocators.h"
 #include "rmw/convert_rcutils_ret_to_rmw_ret.h"
 #include "rmw/error_handling.h"
-#include "rmw/names_and_types.h"
+#include "rmw/event.h"
+#include "rmw/get_node_info_and_types.h"
 #include "rmw/get_service_names_and_types.h"
 #include "rmw/get_topic_names_and_types.h"
-#include "rmw/get_node_info_and_types.h"
-#include "rmw/event.h"
-#include "rmw/validate_node_name.h"
+#include "rmw/incompatible_qos_events_statuses.h"
```

Comment:
@mm318 this should have been conditionally included with a `#if RMW_VERSION_GTE(0, 8, 2)` guard (the header didn't exist before).
Can you open a follow-up?

Reply:
Fixed in https://github.com/ros2/rmw_cyclonedds/pull/127.

Reply:
Sorry, I missed this!

Reply:
@mm318 it happens to us all ... and in this case had a rather positive side-effect: @rotu decided to go set-up GitHub actions to do builds on PRs against dashing and eloquent. I don't know if he has already verified that a build against dashing indeed fails when a >=0.8.2 check is removed, but once we know we can trust it, the risk will have been reduced significantly.